scripts/run_e2e_env.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
+DIR=$(dirname "$0")
```

Comment:
I’ve seen this done like the below before.
```sh
DIR="$(CDPATH= cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
```

It ensures that the directory is obtained correctly even if the script is executed from a different directory or via a symbolic link